src/main/java/step3/Car.java

```diff
@@ -0,0 +1,24 @@
+package step3;
+
+public class Car {
+
+    public Car(int moveNumber) {
+        this.moveNumber = moveNumber;
+    }
+
+    private int distance = 0;
+    private int moveNumber;
+
+    public void move(int randomNum) {
+
+        if(randomNum >= moveNumber) {
+            distance ++;
+        }
+
+    }
```

Comment:
공백은 정말 의미가 구분되어야 하는 경우만 작성해주세요.
```suggestion
    public void move(int randomNumber) {
        if (randomNumber >= moveNumber) {
            distance++;
        }
    }
```